### PR TITLE
Fix sync to Windows deduplication enabled storages  

### PR DESCRIPTION
**Pull request by dragotin**

### CODE REVIEW

**Comment by dragotin** [comment]
@danimo @ogoffart please

File: csync/src/vio/csync_vio_local_win.c [hunk 1]
```diff
@@ -137,40 +137,45 @@ static time_t FileTimeToUnixTime(FILETIME *filetime, DWORD *remainder)
...
+            goto err;
+        }
+    }
+    file_stat->name = c_utf8_from_locale(handle->ffd.cFileName);
 
-  file_stat->fields |= CSYNC_VIO_FILE_STAT_FIELDS_TYPE;
-  if (handle->ffd.dwFileAttributes & FILE_ATTRIBUTE_REPARSE_POINT
-            && handle->ffd.dwReserved0 & IO_REPARSE_TAG_SYMLINK) {
+    file_stat->fields |= CSYNC_VIO_FILE_STAT_FIELDS_TYPE;
+    CSYNC_LOG(CSYNC_LOG_PRIORITY_DEBUG, "WinStat info: %ld - %ld %s", handle->ffd.dwFileAttributes, handle->ffd.dwReserved0, file_stat->name);
```
Comment (ogoffart) [inline csync/src/vio/csync_vio_local_win.c]: how often is this shown?  Will that not spam the log?

**Comment by ogoffart** [comment]
:+1:  if you remove the debug

**Comment by danimo** [comment]
Please add the changelog entry as proposed in https://github.com/owncloud/enterprise/issues/1225?

**Comment by guruz** [comment]
👎 
Please stash the commits. 
It contains unnecessary whitespace changes.
Please mention the public issue number in commit https://github.com/owncloud/client/issues/4056

**Comment by dragotin** [comment]
@guruz @ogoffart @danimo please take a look again. THAT is actually what it should be. I apologize for the mess before.

**Comment by danimo** [comment]
:+1:

**Comment by guruz** [comment]
👍

**Pull request merged**